changelog.md

```diff
@@ -2,3 +2,8 @@
 This file contains all the notable changes done to the Ballerina UDP package through the releases.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+- [[#2197] Add the specification document](https://github.com/ballerina-platform/ballerina-standard-library/issues/2197)
```

Comment:
I think we don't need to update the change log for this.

Reply:
Reverted back.